solutions/task_8.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python3
+"""
+TASK: Implement all needed functions to execute unittests without errors.
+Body of the function can contain only 1 line (usage of ';' is not allowed).
+"""
+import unittest
+from functools import reduce
+
+
+def func1(arr):
+    return sorted(arr, reverse=True)
+
+
+def func2(arr):
+    return [item for sublist in arr for item in sublist]
+
+
+def func3(arr):
+    return [reduce(lambda x, y: x + y, sublist) for sublist in arr]
+
+
+def func4(arr):
+    return [sum([sublist[i] for sublist in arr]) for i in range(4)]
```

Comment:
Overengineering
What if length of sublist will be changed? You will be forced to change hardcoded `4`.